### PR TITLE
Add test for MigrationTask

### PR DESCRIPTION
**Pull request by matthias-ronge**

### CODE REVIEW

File: Kitodo/src/test/java/org/kitodo/production/helper/tasks/MigrationTaskIT.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    @AfterClass
+    public static void cleanDatabase() throws Exception {
+        MockDatabase.stopNode();
+        MockDatabase.cleanDatabase();
+    }
+
+    @Test
+    public void testMigrationTask() throws InterruptedException {
```
Comment (Kathrin-Huber) [inline Kitodo/src/test/java/org/kitodo/production/helper/tasks/MigrationTaskIT.java]: test should test, if metadata is migrated ;-)

Comment (matthias-ronge) [reply Kitodo/src/test/java/org/kitodo/production/helper/tasks/MigrationTaskIT.java]: This is covered by MetsKitodoConverterTest (in Kitodo-DataEditor module)

Comment (Kathrin-Huber) [reply Kitodo/src/test/java/org/kitodo/production/helper/tasks/MigrationTaskIT.java]: More specific: The test should test, if the metadata migration is actually triggered by the task. if I remove everything out of the run() method (but the setProgress100) the test is still green.

File: Kitodo/src/main/java/org/kitodo/production/services/ServiceManager.java [hunk 2]
```diff
@@ -111,6 +111,12 @@ private static void initializeClientService() {
         }
     }
 
+    public static boolean discardDataEditorService() {
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/services/ServiceManager.java]: please do not add a public method for tests

File: Kitodo/src/test/java/org/kitodo/production/helper/tasks/MigrationTaskIT.java (hunk 1 above)
Comment (Kathrin-Huber) [reply Kitodo/src/test/java/org/kitodo/production/helper/tasks/MigrationTaskIT.java]: can't you reuse the test logic from the converter test? A single migration should be enough to test, if everything is triggered alright

**Pull request merged**